src/Selenium2Library/locators/elementfinder.py

```diff
@@ -4,6 +4,25 @@
 from Selenium2Library.utils import escape_xpath_value, events
 
 
+class ParserLocator(object):
+
+    @classmethod
+    def parse_locator(cls, locator):
```

Comment:
1) The ` @classmethod` allows to use the parser method without creating instance, like `LocatorParser.parse(locator)`
2) Agreed and done